sledgehammer/start-up.sh

```diff
@@ -149,6 +149,11 @@ echo "# Sledgehammer added to log to the admin node" >> /etc/rsyslog.conf
 echo "*.* @@${ADMIN_IP}" >> /etc/rsyslog.conf
 service $RSYSLOGSERVICE restart
 
+# Sometimes at this point network is not up yet, wait for it
+ping -c 1 -w 60 $ADMIN_IP > /dev/null || {
```

Comment:
Oh, didn't know that: `With deadline option, ping waits for count ECHO_REPLY packets, until the timeout expires.`